phpstan-baseline.neon

```diff
@@ -570,6 +570,11 @@ parameters:
 			count: 3
 			path: src/Uploadable/UploadableListener.php
 
+		-
+			message: "#^Class Gedmo\\\\Tests\\\\Translatable\\\\Fixture\\\\CategoryTranslation not found\\.$#"
```

Comment:
Baselining this because the YAML mapping that the annotation/attribute mappings were generated from references this non-existing class, and PHPStan now (rightfully) groans about it.  The test fixtures should probably use an existing class at some point in the future, but for now, its absence doesn't break anything.